### PR TITLE
Add Sourcegraph token rotation procedure

### PR DESCRIPTION
**Pull request by shivasurya**
Added sourcegraph token revocation and rotation procedure tutorial

- [x] verified image clarity
- [x] tested locally
- [x] added to index page

### CODE REVIEW

**Comment by shivasurya** [comment]
@dustin-decker Can I get a review please. Thanks

**Review by hxnyk — COMMENTED** [review]
Hey @shivasurya, thanks for contributing! 😄 

Could you arrange the content in this order: generate, replace, and then revoke? We prefer to have the revocation step at the end to minimize any potential impacts on existing services.

**Comment by shivasurya** [comment]
Thanks @hxnyk for your time and review. I have updated the content 👍🏻

File: content/docs/tutorials/sourcegraph.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#### Step 1c. Confirm
+Click the `Generate Token` button to confirm the operation.
+
+![Regenerate access token](/images/sourcegraph/3.png)
+
+## Replace the Leaked Sourcegraph Access token
+Replace the leaked Sourcegraph access token with the new one in all impacted applications and services.
+
+### Revoke the Access Token
```
Comment (hxnyk) [inline content/docs/tutorials/sourcegraph.md:27]: Can you update this to be at heading level 2 to match the generate and replace step?

File: content/docs/tutorials/sourcegraph.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+![Revoke access token](/images/sourcegraph/2.png)
+
+> **Note:** For security reasons, Sourcegraph Access Tokens are masked after creation. If you lose the key, you will need to generate a new one. 
+
+
+---
+
+## Replace the Leaked Sourcegraph Access Token
+Replace the leaked Sourcegraph Access Token with the new one in all impacted applications and services.
```
Comment (hxnyk) [inline content/docs/tutorials/sourcegraph.md:41]: Please delete since moved above to L24-25

**Comment by shivasurya** [comment]
Done 👍🏻

**Review by hxnyk — APPROVED** [review]

**Comment by hxnyk** [comment]
Thanks for the contribution @shivasurya!! 🎉

**Comment by shivasurya** [comment]
Thanks @hxnyk 🎉

**Pull request merged**